src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -550,6 +550,9 @@ private async Task<ConnectBlocksResult> FullyValidateLockedAsync(ChainedHeader n
                     // This might cause uncontrolled memory changes but big reorgs are not common and a chain will anyway get disconnected when the fork is more then 500 blocks.
                     foreach (ChainedHeaderBlock disconnectedBlock in disconnectedBlocks)
                     {
+                        this.logger.LogTrace("[DISCONNECTED_BLOCK_STATE]{0}", disconnectedBlock.ChainedHeader);
+                        this.logger.LogTrace("[DISCONNECTED_BLOCK_STATE]{0}:{1}:{2}", disconnectedBlock.ChainedHeader.Previous);
```

Comment:
I think you copy pasted this and didn't update the interpolation :)

Reply:
Ah yes, will fix.